migration/20221009155225-country-table.js

```diff
@@ -1,6 +1,6 @@
 const { MigrationInterface, QueryRunner } = require("typeorm");
 
-module.exports = class countryTable20221009161225 {
+module.exports = class countryTable20221009155225 {
```

Comment:
You should create a new migration for each change in the DB.
The reason, when you change the name of a migration, that migration will be executed on the machines of other's developers as it is a new migration (with new name). That will cause the starting of the app to fail as the `country` table is already exists.
For now keep it as it it. I will remove the old migration and the table from my DB.
@RamaMahjoub